talpid-core/src/tunnel/openvpn.rs

```diff
@@ -148,6 +158,24 @@ impl<C: OpenVpnBuilder> OpenVpnMonitor<C> {
         let _ = rx.recv().unwrap();
         result
     }
+
+    /// Performs a postmortem analysis to attempt to provide a more detailed error result.
+    fn postmortem(self) -> Result<()> {
+        if let Some(log_path) = self.log_path {
+            if let Ok(log) = fs::read_to_string(log_path) {
+                ensure!(
+                    !log.contains("There are no TAP-Windows adapters on this system"),
```

Comment:
Please tell me that OpenVPN does not translate the error messages to other languages.